packages/components/.storybook/examples/SearchField.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { generateMockOptions } from '../../src/combobox/utils'
+import React, { useState } from 'react'
+import {
+  SearchField,
+  Table,
+  TableHeader,
+  TableBody,
+  Column,
+  Row,
+  Cell,
+} from '../../src/index'
+
+const meta: Meta<typeof SearchField> = {
+  title: 'Examples/Search',
+  tags: ['autodocs'],
+  argTypes: {},
+}
+export default meta
+type Story = StoryObj<typeof SearchField>
+
+type DataRow = { id: number; fruit: string; description: string }
+type ColumnType = {
+  name: string
+  id: string
+  isRowHeader?: boolean
+}
+
+export const SimpleSearch: Story = {
+  args: {},
+
+  render: function SimpleSearchComponent() {
+    const columns: ColumnType[] = [
+      { name: 'Frukt', id: 'fruit', isRowHeader: true },
+      { name: 'Beskrivning', id: 'description' },
+    ]
+
+    const [searchTerm, setSearchTerm] = useState('')
+    const [mockData] = useState<DataRow[]>(() =>
+      Array.from(generateMockOptions(10)).map((item, index) => ({
+        id: index + 1,
+        fruit: item.name,
+        description: `En frukt.`,
+      })),
+    )
+
+    const filteredData = mockData.filter(item =>
+      item.fruit.toLowerCase().includes(searchTerm.toLowerCase()),
+    )
+
+    return (
+      <div style={{ maxWidth: '400px', margin: '0 auto' }}>
+        <SearchField
+          placeholder='Sök efter en frukt...'
+          buttonText='Sök'
+          onInput={e => setSearchTerm((e.target as HTMLInputElement).value)}
```

Comment:
```suggestion
          onChange={value => setSearchTerm(value)}
```
Jag skulle föreslå att du använder denna `onChange` istället för `onInput`, då fångar du även om om användaren klickar på rensa-knappen med musen